Calibration/EcalCalibAlgos/interface/ElectronRecalibSuperClusterAssociator.h

```diff
@@ -7,7 +7,7 @@
 //
 // Description:
 
-#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/one/EDProducer.h"
```

Comment:
Is there a specific need that this is a `one` module? Could it be `stream` instead?

Reply:
No specific need - I've just updated to `stream` which required an adjustment of the `ECALpedestalPCLworker` class defined in `Calibration/EcalCalibAlgos/interface/ECALpedestalPCLworker.h` (not sure why)

Reply:
Indeed it compiles following that suggestion. Interesting problem.